### PR TITLE
[Java shade client] Add encryption integration test

### PR DESCRIPTION
**Pull request by RobertIndie**
### Motivation
Currently, there are no encryption integration test cases in the shaded client test.
#6834 needs more test cases to address the issue.

### Modification
Add encryption integration test cases to the shaded client test.

### CODE REVIEW

File: tests/pulsar-client-all-shade-test/src/test/java/org/apache/pulsar/tests/integration/SimpleProducerConsumerTest.java [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+public class SimpleProducerConsumerTest {
+    private static final Logger log = LoggerFactory.getLogger(SimpleProducerConsumerTest.class);
+
+    private PulsarContainer pulsarContainer;
+    private URI lookupUrl;
+    private PulsarClient pulsarClient;
+
+    @BeforeClass
+    public void setup() throws PulsarClientException, URISyntaxException, PulsarAdminException {
+        pulsarContainer = new PulsarContainer();
```
Comment (eolivelli) [inline tests/pulsar-client-all-shade-test/src/test/java/org/apache/pulsar/tests/integration/SimpleProducerConsumerTest.java:66]: Shall we dispose this object?

Comment (RobertIndie) [reply tests/pulsar-client-all-shade-test/src/test/java/org/apache/pulsar/tests/integration/SimpleProducerConsumerTest.java:58]: Yes. Thanks.

**Review by sijie — APPROVED** [review]

**Pull request merged**